### PR TITLE
added partners fields

### PR DESCRIPTION
**Pull request by 4gnle**

### CODE REVIEW

**Comment by adiwajshing** [comment]
@angeljgomezc I hope you didn't edit the generated types file directly. It says at the top:
```
/**
 * This file was auto-generated by openapi-typescript.
 * Do not make direct changes to the file.
 */
```

Also, this is missing the changes required in the openapi file

**Comment by 4gnle** [comment]
@adiwajshing re-did this - check and let me know!

File: openapi.yaml [hunk 1]
```diff
@@ -376,6 +395,7 @@ components:
       additionalProperties: false
       required:
         - title
+        - bio
```
Comment (adiwajshing) [inline openapi.yaml]: If this is made required, it'll break all existing grants -- will need to remove that

Comment (4gnle) [reply openapi.yaml]: fixing!

File: openapi.yaml [hunk 2]
```diff
@@ -385,6 +405,9 @@ components:
         title:
           type: string
           maxLength: 128
+        bio:
```
Comment (adiwajshing) [inline openapi.yaml:410]: bio & partners should be available in WorkspaceCreateRequest & WorkspaceUpdateRequest

**Comment by adiwajshing** [comment]
@angeljgomezc almost there -- just need to add two test cases:
1. workspace create with bio & partners
2. workspace update with bio & partners

File: openapi.yaml (hunk 2 above)
Comment (angeljgomezc) [reply openapi.yaml:410]: > @angeljgomezc almost there -- just need to add two test cases:
> 
> 1. workspace create with bio & partners
> 2. workspace update with bio & partners

can you point me where? or is it done already?

**Comment by 4gnle** [comment]
> @angeljgomezc almost there -- just need to add two test cases:
> 
> 1. workspace create with bio & partners
> 2. workspace update with bio & partners

can you point me where? or is it done automatically?

**Comment by adiwajshing** [comment]
@angeljgomezc need to add test case here: https://github.com/questbook/service-validator/blob/main/src/tests/test.validation.ts

**Review by adiwajshing — APPROVED** [review]

**Pull request merged**